src/utils/urlParseHelper.ts

```diff
@@ -15,15 +15,20 @@ export interface IParsedUrl {
  * @returns Object containing scmRepository URL, parsed URL and display name representing URL
  *  */
 export const parseInternalScmRepositoryUrl = ({ url }: IScmRepositoryUrl): IParsedUrl => {
-  const protocol = url.includes('git@') ? 'git@' : url.split('://')[0];
-  const base = url.includes('git@') ? url.split('@').at(1)?.split(':').at(0) : url.split('://')[1].split('/')[0];
-  const project = url.includes('git@')
+  const gitlabBase = 'gitlab.cee.redhat.com';
+  const isGitProtocol = url.includes('git@');
+  const protocol = isGitProtocol ? 'git@' : url.split('://')[0];
+  const base = isGitProtocol ? url.split('@').at(1)?.split(':').at(0) : url.split('://')[1].split('/')[0];
+  const project = isGitProtocol
     ? url.split(':').at(1)
+    : url.includes(gitlabBase)
+    ? url.split(gitlabBase + '/').at(1)
     : url.split(base + (['https', 'http'].includes(protocol) ? '/gerrit/' : '/'))[1];
```

Comment:
```ts
const project = isGitProtocol
    ? url.split(':').at(1)
    : url.split(base + url.includes(gitlabBase)  ? '/' : '/gerrit/')).at(1);
```

Reply:
This refactored code dose not have the same logic as the original one and will cause failure of parsing the URL start with `https://gitlab.***.com/` without `/gerrit/`. I will keep the original logic but with all [] change to at().